content/10-std.md

```diff
@@ -996,8 +999,28 @@ All spawned threads are treated as daemon threads, meaning that the main thread
 
 Due to threads having access to a shared memory space with all the Haxe variables and objects, it is possible to run into issues due to deadlocks and race conditions. The standard library provides some core synchronization constructs in the [`sys.thread`](https://api.haxe.org/sys/thread/) package.
 
+<!--label:std-sys-streams-->
+#### Streams
```

Comment:
I think "Standard IO Streams" is a better title for this section as "streams" usually refers to a generic stream API.